### DISCUSSION TITLE
which 32 bit version to download for directx renderer NO qt?

### DISCUSSION THREAD

**Discussion by honx4amiga** [86Box/86Box#2473] [Q&A] [answered]
it's been a while since i last downloaded a newer build of 86box (still using build #3181). now i'm reading that newer builds use qt (which causes very high cpu load which i definitely DON'T want) and qt builds don't have directx renderer anymore (which is also bad). so which 32 bit file should i download which is NOT infested with cpu-eating qt and instead still offers directx as renderer?

**Comment by dhrdlicka** [comment, marked as answer]
Latest builds do include a new Direct3D 9 renderer. As for the performance issues, it might be better to try the new builds out and see for yourself rather than be scared by bug reports from highly specific hardware configurations. Most users don't experience any significant performance changes between Qt and the old UI.

**Reply by TC1995** [reply]
Try the latest builds of 86box, there have been massive updates to the perfomance.

**Reply by OBattler** [reply]
Yes, but they haven't done (and can't do) anything about QT's CPU usage.
